src/main/pages/setup-docker/docker-cli.adoc

```diff
@@ -8,142 +8,216 @@ folder: setup-docker
 
-Used to download Docker images that will be stored in your Docker images repository. This command downloads images that are used by the CLI as utilities, for Che to do initialization and configuration, and for the runtime images that Che needs when it starts. This command respects `--offline`, `--pull`, `--force`, and `--no-force` (default). This command is invoked by `che init`, `che config`, and `che start`.
+It respects `--no-force`, `--pull`, `--force`, and `--offline` command options.
 
-`download` is invoked by `che init` before initialization to download images for the version specified by `eclipse/che:<version>`.
+*_destroy_*
 
-It is possible to override the docker images used by the CLI by setting the following environment variables:
+The `destroy` command deletes the `/docs`, `che.env` and `/instance` directories, including user workspaces, projects, data, and user database. To skip the confirmation warning message, pass the `--quiet` argument in the command. To delete the `cli.log` file, pass the `--cli` argument. By default, the `cli-log` file is retained for traceability.
 
-* `IMAGE_INIT` to override the default `eclipse/che-init:<version>` docker image,
+*_dir_*
+
+The `dir` command boots a new Eclipse Che instance with a workspace for the `:/chedir` directory defined as volume mount in the parameter.
+
+For example, if you give `$HOME/my-project` as a parameter, a new Che instance is created using `$HOME/my-project` as a project in the IDE. Inside the IDE, the `/projects` directory contains a `my-project` directory with your host directory. Any changes inside the IDE are reflected in your host directory. And, updating a file on your local computer updates the content of the file inside the IDE.
+
+* `init`: Initializes the directory specified and adds a default `Chefile` file if there is none.
+* `up`: Boots Eclipse Che with workspace on directory.
+* `down`: Stops Eclipse Che and any workspaces.
+* `ssh`: Connects to the running workspace by using the `ssh` command.
+* `status`: Displays if an instance of Eclipse Che is running or not for the specified directory.
+
+*_download_*
+
+The `download` command is used to download Docker images stored in your Docker images repository. This command downloads images that are used by the CLI as utilities, for Che to do initialization and configuration, and for the runtime images that Che needs when it starts. This command respects `--offline`, `--pull`, `--force`, and `--no-force` (default) command options. It is invoked by the `che init`, `che config`, or `che start` commands.
+
+The `download` command is invoked by the `che init` command before initialization to download images for the version specified by `eclipse/che:<version>`.
+
+To override the docker images used by the CLI, set the following environment variables:
```

Comment:
Docker images. Also, as mentioned before, do we actually mean Docker-formatted container images?